pykechain/models/property_multi_reference.py

```diff
@@ -39,11 +39,13 @@ def _retrieve_objects(self, **kwargs) -> List[Part]:
                 else:
                     models = self.model().value
                 if models:
-                    parts = list(self._client.parts(
-                        id__in=','.join(part_ids),
-                        model=models[0],
-                        category=None,
-                    ))
+                    parts = list()
+                    for chunk in self.chunks(part_ids):
+                        parts += list(self._client.parts(
```

Comment:
`parts.extend()` is faster than concatenation. 

```suggestion
                        parts.extend(
                            list(self._client.parts(
```